src/components/ImageMapEditor/ImageMapEditor.tsx

```diff
@@ -0,0 +1,197 @@
+import { useState } from "react";
+import { Rnd } from "react-rnd";
+import ReactCrop from "react-image-crop";
+import type { Crop } from "react-image-crop";
+import { AiOutlinePicture, AiOutlineDelete } from "react-icons/ai";
+import type { Coordinate } from "types/coordinate";
+import {
+  ImageMapEditorContainer,
+  ImageMapEditorHeader,
+  ImageMapEditorHeaderCircle,
+  ImageMapEditorBody,
+  ImageMapEditorInput,
+  ImageMapEditorInputPlaceholder,
+  ImageMapEditorUploader,
+  ImageMapEditorImagePreview,
+  ImageMapEditorNumberTag,
+  ImageMapEditorUploaderDeleteButton,
+} from "./ImageMapEditor.style";
+
+import "react-image-crop/dist/ReactCrop.css";
+
+const CROP_BASE_COLOR = "#1070e0";
+const CROP_RESIZE_BASE_SIZE = 8;
+
+const rndBaseStyle = {
+  display: "flex",
+  padding: 5,
+  border: `solid 1px ${CROP_BASE_COLOR}`,
```

Comment:
Why not like the rest of the `border` as `1px solid ${CROP_BASE_COLOR}`?

Reply:
it should follow the border shorthand property. I’ll change the shorthand property of position.

Reply:
fixed in https://github.com/chatbotgang/image-map-editor/pull/30/commits/06dea58f14525f33f6c928c0dff8b77cc6e23588